Automattic/wp-calypso
# Gutenboarding: convert design screenshots to webp

## Changes proposed in this Pull Request

To use `webp` format for images in the Gutenboarding design selector.

Discussion in p1594723857059700-slack-CNMU1UHHB

## Testing instructions

Take a look at the screenshots at the `/new/design` step. 

How do they look?

Now check in IE11. Do the JPEGs load instead?
